2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py

```diff
@@ -0,0 +1,22 @@
+# made by vanguard7
+def main():
+    t = int(input())
+    for i in range(t):
+        n = int(input())
+        low = 10000000000000000000000000000000000000000000
+        index = -1
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpdjftdys7/2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py
+++ b/tmp/tmpdjftdys7/2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py
@@ -19,4 +19,4 @@
 
 if __name__ == '__main__':
     main()
-    
+
```

Reply:
@vangurad07 Its trailing whitespaces at end of line.